### PR TITLE
cooler combat sound effects

### PR DESCRIPTION
**Pull request by tyeagg**
## About The Pull Request
makes some sounds cooler, explosion sounds are now different if its devastating or heavy or light
got sounds from this foxhole mod thats cool https://liquidpopsicle.itch.io/liquid-sound-overhaul

https://user-images.githubusercontent.com/98577688/219667363-838166d2-1d09-487c-bebd-974a6067569d.mp4

https://user-images.githubusercontent.com/98577688/219666852-7778c911-e5bf-4ff3-ad1d-6f95f9174ac0.mp4
## Why It's Good For The Game
sounds gooder
## Changelog
:cl:
add: different sounds for different explosion types
soundadd: new explosion sounds
soundadd: new bullet ricochet and miss
sounddel: removed old explosion sounds
/:cl:

### CODE REVIEW

**Comment by yyzsong** [comment]
Finally, xenos can have the true Somme experience they've all been waiting for. Thank you tyeagg

**Comment by Mannybrado** [comment]
new hit sounds kind of suck
other ones are cool

**Comment by tyeagg** [comment]
> new hit sounds kind of suck
> other ones are cool

there are no changes to mob hit sound thats just the sound when bullets dont penetrate the armor on mobs

File: code/game/sound.dm [hunk 1]
```diff
@@ -187,17 +191,13 @@ A good representation is: 'byond applies a volume reduction to the sound every X
...
 		if("ballistic_bounce")
 			S = pick('sound/bullets/bullet_ricochet1.ogg','sound/bullets/bullet_ricochet2.ogg','sound/bullets/bullet_ricochet3.ogg','sound/bullets/bullet_ricochet4.ogg','sound/bullets/bullet_ricochet5.ogg','sound/bullets/bullet_ricochet6.ogg','sound/bullets/bullet_ricochet7.ogg','sound/bullets/bullet_ricochet8.ogg')
 		if("rocket_bounce")
 			S = pick('sound/bullets/rocket_ricochet1.ogg','sound/bullets/rocket_ricochet2.ogg','sound/bullets/rocket_ricochet3.ogg')
 		if("energy_hit")
 			S = pick('sound/bullets/energy_impact1.ogg')
-		if("energy_miss")
-			S = pick('sound/bullets/energy_miss1.ogg')
-		if("energy_bounce")
-			S = pick('sound/bullets/energy_ricochet1.ogg')
```
Comment (TiviPlus) [inline code/game/sound.dm:200]: why remove this?

**Review by TiviPlus — APPROVED** [review]
rest lgtm

**Comment by Lumipharon** [comment]
I like the mid/large explosion sounds, but the small explosions sound kinda weird coming from grenades.

Also can you keep in the existing small explosion sounds? They are used for stuff like airburst rounds and I specifically don't want them to sound like actual explosives.

**Review by Lumipharon — APPROVED** [review]

File: code/modules/projectiles/projectile.dm [hunk 2]
```diff
@@ -1296,7 +1299,11 @@ So if we are on the 32th absolute pixel coordinate we are on tile 1, but if we a
 	if(!P.ammo.ping)
 		return
 	if(prob(65))
-		if(P.ammo.sound_bounce) playsound(src, P.ammo.sound_bounce, 50, 1)
+		var/pitch = 0
+		if(P.ammo.flags_ammo_behavior & AMMO_SOUND_PITCH)
+			pitch = 55000
```
Comment (Lumipharon) [inline code/modules/projectiles/projectile.dm:1304]: This shouldn't run in the ammo doesn't have the relevant sound to begin with. Also you could probs just chuck it in the playsound line itself.

**Review by Lumipharon — CHANGES_REQUESTED** [review]

**Review by Lumipharon — APPROVED** [review]

**Pull request merged**